pavelib/quality.py

```diff
@@ -858,6 +858,42 @@ def run_pii_check(options):
         fail_quality('pii', full_log)
 
 
+@task
+@needs('pavelib.prereqs.install_python_prereqs')
+@timed
+def check_keywords():
+    """
+    Check Django model fields for names that conflict with a list of reserved keywords
+    """
+    report_path = os.path.join(Env.REPORT_DIR, 'reserved_keywords')
+    sh("mkdir -p {}".format(report_path))
+
+    overall_status = True
+    for env, env_settings_file in [('lms', 'lms.envs.test'), ('cms', 'cms.envs.test')]:
+        report_file = "{}_reserved_keyword_report.csv".format(env)
+        try:
+            sh(
+                "export DJANGO_SETTINGS_MODULE={}; "
+                "python manage.py {} check_reserved_keywords "
+                "--override_file {}/db_keyword_overrides.yml "
+                "--report_path {} "
+                "--report_file {} "
+                "--verbosity 2 ".format(
```

Comment:
Did you mean to leave verbosity at 2 (1 is the default)?

Reply:
another Nit: instead of one long string can you maybe break this up into a list of tokens, or instead name the template tokens? Also using os.path.join for the --override_file argument seems more consistent with the rest of the code.